app.py

```diff
@@ -55,16 +56,25 @@ def populate_cfg_by_env(cfg):
     env_model_settings['access_key_id'] = os.environ.get('S3_ACCESS_KEY')
     env_model_settings['secret_access_key'] = os.environ.get('S3_SECRET_KEY')
 
+    env_common_settings = {}
+    env_common_settings['credentials'] = \
+        json.loads(os.environ.get('SERVICE_CREDENTIALS'))
```

Comment:
The option (env variable) is mandatory, right? If so, I think, it is correct. However I would additionally check that the json contains at least one user+password pair.

Reply:
Add additionally check.